src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetController.java

```diff
@@ -1,4 +1,49 @@
 package uk.gov.companieshouse.web.accounts.controller.smallfull;
 
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+
+@Controller
 public class BalanceSheetController {
-}
+
+    private static final String BALANCE_SHEET_PATH = "/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/balance-sheet";
```

Comment:
So, back in Abridged, these URLs where flagged by Sonar because they were pasted in each controller. Could we have a function that creates these URLs dynamically? I'm only saving you the hassle of when Sonar raises it :(

Reply:
We're looking into implementing an interface in each controller to construct previous / next paths to redirect between each page of the journey so we'll look to resolve it there. Cheers for raising it 👍

Reply:
👍